### PR TITLE
Upgraded github-api dependency

### PR DESCRIPTION
**Pull request by recena**
We need to cut a release of this plugin including the latest release of `github-api` library.

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   </licenses>
 
   <artifactId>github-api</artifactId>
-  <version>1.72.2-SNAPSHOT</version>
+  <version>1.75-SNAPSHOT</version>
```
Comment (tfennelly) [inline pom.xml]: `SNAPSHOT` ? The second change is without.

Comment (recena) [inline pom.xml]: OMG! Where is my head!?!?! Thanks!

Comment (recena) [inline pom.xml]: Ah no. It is fine because this is dev branch. This should be changed by `mvn release` process.

File: pom.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
     <dependency>
       <groupId>org.kohsuke</groupId>
       <artifactId>github-api</artifactId>
-      <version>1.72</version>
+      <version>1.75</version>
```
Comment (recena) [inline pom.xml]: @tfennelly I was confused about this.

**Comment by tfennelly** [comment]
🐝

Assuming you have the version numbers correct :)

**Comment by svanoort** [comment]
🐝  - this is correct, since plugin tracks the version of the github API but releases separately

**Comment by recena** [comment]
@svanoort @tfennelly Thanks.

**Comment by recena** [comment]
@reviewbybees done

File: pom.xml (hunk 1 above)
Comment (amuniz) [inline pom.xml]: Why are you jumping from 1.72.2 to 1.75? Does it always match the `github-api` version?

Comment (recena) [inline pom.xml]: @amuniz There is a relation between `github-api-plugin` and `github-api` (library).

Comment (KostyaSha) [inline pom.xml]: @amuniz yes, it always matches.

**Comment by KostyaSha** [comment]
I was caring about this library update and tests last year, when nobody supported it. Now CloudBees doing any updates they want to do. Nice cooperation.

**Pull request merged**